lib/bike.dart

```diff
@@ -46,18 +46,25 @@ class Bike extends _$Bike {
       return;
     }
     if (_updateDebounce?.isActive ?? false) _updateDebounce?.cancel();
-    _updateDebounce = Timer(const Duration(seconds: 2), () async {
-      var data = await ref
+    _updateDebounce = Timer(const Duration(milliseconds: 1500), () async {
+      var RIDE = await ref
           .read(bluetoothRepositoryProvider)
-          .readCurrentState(state.id);
-      if (data == null || data.isEmpty) {
+          .readCurrentState(state.id, [2, 3]);
+      var MOTION = await ref
+          .read(bluetoothRepositoryProvider)
+          .readCurrentState(state.id, [2, 1]);
+      var SETTINGS = await ref
+          .read(bluetoothRepositoryProvider)
+          .readCurrentState(state.id, [3, 0]);
+      if (SETTINGS == null || SETTINGS.isEmpty || RIDE == null || RIDE.isEmpty || MOTION == null || MOTION.isEmpty) {
```

Comment:
Here you can do something like:
```dart
if ([SETTINGS, RIDE, MOTION].any(e => e == null || e.isEmpty)){
return;
}
```
It's nice since all the logic is in one place.

Reply:
On the other hand, this is the only place that calls `readCurrentState`. It might be better to move all this logic into that method as these state IDs probably shouldn't be known outside the Bluetooth repository. It's a low level implementation detail.

Reply:
> Here you can do something like:
> 
> ```dart
> if ([SETTINGS, RIDE, MOTION].any(e => e == null || e.isEmpty)){
> return;
> }
> ```
> 
> It's nice since all the logic is in one place.

Dart, gave me red lines, when attempting to apply this method.